src/session.c

```diff
@@ -107,7 +107,8 @@ const char *session_error_message[] = {
 // cgi.c
 extern int headers_initialized;
 extern void libcgi_error(int error_code, const char *msg, ...);
-extern formvars *process_data(char *query, formvars **start, formvars **last, const char delim, const char sep);
+extern formvars *process_data(char *query, formvars **start, formvars **last,
+                              const char sep_value, const char sep_name);
 
```

Comment:
Declaration and definition have different signatures. One is 

`formvars *process_data(const char *query, formvars **start, formvars **last, const char sep_value, const char sep_name)`

while the other is

`formvars *process_data(char *query, formvars **start, formvars **last, const char sep_value, const char sep_name)`.